container.go

```diff
@@ -0,0 +1,31 @@
+package containerd
+
+import "golang.org/x/net/context"
+
+type Container interface {
+	// ID of the container
+	ID() string
```

Comment:
Rather than have a method for each field, would it be better to collect them into a metadata or info type that has the runtime and id on it? I have been toying with interface design in this area to avoid method bloat for what should just be a field.